plutus-core/cost-model/data/builtinCostModelA.json

```diff
@@ -698,6 +698,26 @@
             "type": "constant_cost"
         }
     },
+    "lengthArray" : {
+        "cpu": {
+            "arguments": 99999999999999,
```

Comment:
Temporary value (stub)

Reply:
>  @kwxm should we add this stuff when adding a new builtin before costing is done?

I think you have to add _something_ here once you've added `paramLengthArray` [here](https://github.com/IntersectMBO/plutus/blob/5de42d2aa3c1729c27540e9feced4f0408a5169e/plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/BuiltinCostModel.hs#L131), and similarly for the other new functions.  However, if you use `unimplementedCostingFunction` in `toBuiltinMeaning`, these numbers won't be used: they only come into play when you tell the denotation to run the real costing function.  I'd normally add a new builtin without adding any costing code at all in order to test that it behaves as it's supposed to, and then do all of the costing at once. That's just to cut down on the initial amount of work, and the way all the costing code is plumbed in it's probably not obvious that you don't initially need it.  The important thing is that there's no link at all between the builtin and the costing code until you put `runCostingFunction<N>Arguments . param<MyNewBuiltin>` in `toBuiltinMeaning`.  Maybe we should try to make that fact clearer to implementers somewhere.

Reply:
Right, doing it first time it isn't 100% clear where to draw the line, so I followed the compiler and it wanted me to fix JSON as there is TH which reads it and fails to compile if the section is missing.